code/ai/aibig.cpp

```diff
@@ -457,13 +456,46 @@ int ai_big_maybe_follow_subsys_path(int do_dot_check)
 		if ( aip->path_length > 0 ) {
 			int path_done=0;
 			int in_view=0;
+			int default_dist = 5;
 
 			Assert(aip->path_length >= 2);
 			dist = vm_vec_dist_quick(&Path_points[aip->path_start+aip->path_length-2].pos, &Pl_objp->pos);
 
+			// distance setting --wookieejedi
 			if ( aip->path_cur >= (aip->path_start+aip->path_length-1) ) {
+				// register that path is done and reset goal distance to default
+				aip->path_goal_dist = default_dist;
 				path_done=1;
 			}
+			// if not done then set path distances based on specified options
+			else {
+				bool found_point = false;
+				// 1. use radii of path points if specified
+				if (aip->ai_profile_flags[AI::Profile_Flags::Use_subsystem_path_point_radii]) {
+					// need a valid path and point in model to use pof specified radius
+					int path_num = aip->targeted_subsys->system_info->path_num;
+					if ((0 <= path_num) && (path_num <= pm_t->n_paths)) {
```

Comment:
Should this be `< n_paths`, not `<=`?

Reply:
Ah correct, it should be since indexing starts at 0 not 1.